gallery/index.md

```diff
@@ -184,6 +184,11 @@ Here are many fine games made using PlayN. Click any image for more details.
     <img src="collaborationstation/thumb.jpg">
     <span class="thumb-text"><span>Collaboration Station</span></span>
   </a></li>
+
+  <li><a href="tnmt">
+    <img src="tnmt/thumb.jpg">
+    <span class="thumb-text"><span>Traveler's Notebook: Monster Tales</span></span>
```

Comment:
You committed thumb.png but reference thumb.jpg. Best would be to reencode the png as jpg to keep it small, but if not, then switch the URL to png. Thanks!